pkg/processor/runtime/python/py/test_wrapper.py

```diff
@@ -96,7 +110,14 @@ def test_non_utf8_headers(self):
         self._wait_until_received_messages(expected_messages)
 
         malformed_response = self._unix_stream_server._messages[-3]['body']
-        self.assertEqual(http.client.INTERNAL_SERVER_ERROR, malformed_response['status_code'])
+
+        # when using raw, the "malformed" is actually considered valid, as msgpack
+        # being able to deserialize non utf-8 event messages.
```

Comment:
explain in docstring and also mention in `# middle event is malformed` that the behavior is conditioned on whether this tries to decode the string or not

commented somewhere else that imo "decode events" -> "decode strings"